light-clients/icsxx-cf-solana/src/solana/shred/legacy.rs

```diff
@@ -0,0 +1,263 @@
+//! File source: solana/ledger/src/shred/legacy.rs
```

Comment:
I’m guessing we can get rid of legacy support?

Reply:
It's not being use in Solana anymore? Then I think it's crucial to make sure that the encoding is till correct, which seems a bit subtle to me... But I will add a TODO for it

Reply:
Honestly I don’t know, but that would be my assumption. It’s probably something to verify.